portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -22,12 +22,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     UserService userService = UserServiceFactory.getUserService();
     PrintWriter writer = response.getWriter();
 
+    boolean isLoggedIn = userService.isUserLoggedIn();
+    response.setContentType("text/html");
+    writer.println("false");
+
     if (userService.isUserLoggedIn()) {
-      writer.println("The user is logged in!");
+      // writer.println("The user is logged in!");
```

Comment:
Delete these and  replace with the above:
writer.println("false");
in both cases (and remove the original line). Add // TODO comments explaining the propper logic will be added later.